src/rgw/rgw_cr_rados.cc

```diff
@@ -283,6 +283,7 @@ int RGWRadosGetOmapKeysCR::send_request() {
   op.omap_get_vals2(marker, max_entries, entries, nullptr, &rval);
 
   cn = stack->create_completion_notifier();
+  cn->get();
```

Comment:
was this meant to go in `RGWRadosRemoveOmapKeysCR::send_request()`?

it looks like this reference on `cn` is missing from a lot of these coroutines. could you please go through and change these raw `RGWAioCompletionNotifier*` pointers to `boost::intrusive_ptr<RGWAioCompletionNotifier>`? see `RGWRadosRemoveCR` as an example